.gitignore

```diff
@@ -1,3 +1,4 @@
 build
 
 site/
+docker/**/docker-compose.*.yml
```

Comment:
Why `docker/**/...` and not `docker/*/...` - is this some kind of special syntax of `.gitignore` to have two asterisk?

Reply:
Also - IMHO it is confusing to see something like this in .gitignore. I understand why it is here because I read the doc, but just by inspecting the code it would be hard to figure out.

Do we have to put this into ignore, even if we do not currently have files like this in the repo?

I would prefer that we don't, and we document to the piple who create `docker/docker-compose.customX.yml` files how to put them into the their `.gitignore` themselves